alica_engine/include/engine/blackboard/KeyMapping.h

```diff
@@ -1,22 +1,29 @@
 #pragma once
 
 #include <string>
+#include <variant>
 #include <vector>
 
 namespace alica
 {
 
 class Blackboard;
 
+struct Mapping
+{
+    std::variant<std::string, std::string> parentKeyOrConstant;
```

Comment:
imo we should define real types for each of these (i.e. a simple struct constructable for string) so that we can use the type to distinguish the argument

Reply:
okay, makes sense